library/src/main/res/values/attrs.xml

```diff
@@ -2,6 +2,7 @@
 <resources>
     <declare-styleable name="DonutProgressView">
         <attr name="donut_strokeWidth" format="dimension"/>
+        <attr name="donut_strokeCap" format="string"/>
```

Comment:
I think enum would be more appropriate format here. With enum, you get a nice autocomplete for available options in layout editor. Take a look at #58, it's implemented there.

Reply:
Yeah, that is a good idea. I will change it.